docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
yeah, exactly, if you are using I13nMixin, we can access the i13nNode by accessing `this._i13nNode`, but I think it's even better to have a function like `this.getI13nNode()` 

I13Utils is a subset of I13nMixin, so using I13nMixin can still get that function